src/parse-stream.js

```diff
@@ -325,6 +325,18 @@ export default class ParseStream extends Stream {
       });
       return;
     }
+    match = (/^#EXT-X-PROGRAM-DATE-TIME:?((\d{4})-((0[1-9])|(1[0-2]))-((0[1-9])|(([1-2]\d)|(3[0-1])))T(([0-1]\d)|(2[0-3])):([0-5]\d):([0-5]\d)(\.(\d{1,3}))?((\+|-)((0\d)|(1[0-2])):([0-5]\d))?)?$/).exec(line);
+    if (match) {
+      event = {
+        type: 'tag',
+        tagType: 'date-time'
+      };
+      if (match[1]) {
+        event.dateTime = match[1];
```

Comment:
I store the ISO formatted date string here so that the user has the option of using it as they will. When this string is used to create a `Date` object, it gets translated into the users locale and loses reference to the timezone of the source media. Users may want to be able to reference this information for their own purposes.